index.html

```diff
@@ -2,16 +2,12 @@
 
 <html>
 <head>
-  <meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>
+  <meta charset="UTF-8"/>
```

Comment:
See [this comment](https://github.com/octocat/Spoon-Knife/pull/23214#issue-633736235) for why I changed that.